extensions/wikia/UserLogin/UserLoginForm.class.php

```diff
@@ -52,24 +52,30 @@ function load() {
 		$this->wpUserBirthDay = strtotime( $this->wpBirthYear . '-' . $this->wpBirthMonth . '-' . $this->wpBirthDay );
 	}
 
-	// add new account
+	/**
+	 * Adds a new user account and sends a confirmation email.
+	 *
+	 * @return bool|User an instance of User on success; boolean false otherwise.
```

Comment:
It does make sense to return `null` instead of `false` on failure, but it implies considerable refactoring effort as most if not all methods called within the two return `false`. Taking the time pressure into account, `false` seems good enough.